.github/workflows/storage-report.yml

```diff
@@ -0,0 +1,39 @@
+name: Check Storage Layout
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - "dev"
+jobs:
+  check_storage:
+    runs-on: "ubuntu-latest"
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive
+      - name: Install Foundry
+        uses: foundry-rs/foundry-toolchain@v1
+        with:
+          version: nightly
+
+      - name: "Generate and prepare the contract artifacts for current branch"
+        run: |
+          bash bin/storage-report.sh
+
+      - name: Checkout main
+        env:
+          MAIN: ${{ github.event.pull_request.base.sha }}
+        run: |
+          git fetch origin $MAIN
+          git checkout $MAIN
+
+      - name: "Generate and prepare the contract artifacts for main branch"
+        run: |
+          bash bin/storage-report.sh
+
+      - name: Compare outputs
+        run: |
```

Comment:
In my workflow I was outputting to two separate directories and the diffing the contents of the two directories

Reply:
Nice catch, will fix.